client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -140,6 +140,7 @@ const PersonalPurchase = ( {
 				<StatsPWYWUpgradeSlider
 					settings={ sliderSettings }
 					currencyCode={ currencyCode }
+					defaultStartingValue={ defaultStartingValue }
```

Comment:
I was actually expecting the value would come from an API and based on real data but if we're just pretending this is an average, I guess we could set up a constant and work from there. The current fraction is actually `0.5` though. So $7 USD of a possible $14. I'll have another look at this during my testing this week.

Reply:
I meant the original static amount for USD was $7, and the maximum price in USD is $10. However, the current design shows $5 for the average amount. So, maybe we could apply `0.5` for now.

Reply:
Since this is a change, could we discuss this in the team channel to raise the visibility and loop in the designers to verify if this was a desired change or just a placeholder number?